### ISSUE TITLE
Sanitize link attribute 

### ISSUE THREAD

**Issue by stonecold123** [nozer/quill-delta-to-html#15]
I've found that people can inject xss in the link attribute:
````
{"ops":[{"attributes":{"link":"javascript:alert(8007)"},"insert":"link"}]}
````
The browser will execute the javascript when you click the link.

Would you consider removing this sort of injection in the link attribute by escaping these characters?

````
export function escapeLink(string) {
    return string
        .replace(/&/g, "&amp;")
        .replace(/</g, "&lt;")
        .replace(/>/g, "&gt;")
        .replace(/\(/g, "&#40;")
        .replace(/\)/g, "&#41;")
        .replace(/"/g, "&quot;");
};
````

**Comment by nozer** [comment]
Will do. thank you for the contribution

**Comment by nozer** [comment]
Fixed and pushed to tag `v0.7.0`
